api/src/main/java/uk/gov/dhsc/htbhf/claimant/communications/EmailMessagePayloadFactory.java

```diff
@@ -32,8 +33,20 @@ public static String formatPaymentAmountSummary(String summaryTemplate, int numb
         return formatPaymentAmount(summaryTemplate, numberOfVouchers, voucherAmountInPence);
     }
 
-    public static String formatRegularPaymentAmount(int numberOfVouchers, int voucherAmountInPence) {
```

Comment:
This one line method was only called in one place so it's been inlined to reduce the number of methods on the class.